TestFoundation/TestIOStream.swift

```diff
@@ -0,0 +1,422 @@
+// This source file is part of the Swift.org open source project
```

Comment:
There are a lot of overlapping tests from Value to Ref types not sure if I should delete these